db/migrate/20210616103631_create_npq_profiles.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class CreateNpqProfiles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :npq_profiles do |t|
+      t.references :user
+      t.references :npq_lead_provider
+      t.references :npq_course
+
+      t.date :date_of_birth
+      t.text :teacher_reference_number
+      t.boolean :teacher_reference_number_verified, default: false
+      t.text :school_urn
+      t.text :headteacher_status # no, yes_in_first_two_years, yes_over_two_years
```

Comment:
Can this be defined as an enum on the model?

Reply:
I've added the enum in, i've kept them as strings